frontend/main.js

```diff
@@ -710,14 +711,20 @@ function processUpdate(data, init) {
                                 '<div class="arrow still-' + shares[user].state + '" id="arrow-' + shares[user].id + '"></div>' +
                                 '<p class="' + shares[user].state + '" id="label-' + shares[user].id + '">' +
                                     '<span id="nickname-' + shares[user].id + '"></span>' +
-                                    '<span class="velocity">' +
+                                    '<span class="metric">' +
                                         '<span id="velocity-' + shares[user].id + '">0.0</span> ' +
                                         VELOCITY_UNIT.unit +
-                                    '</span><span class="offline" id="last-seen-' + shares[user].id + '">' +
+                                    ' | </span>' + 
+                                    '<span class="metric">' +
+                                        '<span id="altitude-' + shares[user].id + '">0.0</span> ' +
```

Comment:
Would probably feel a bit more compact if we add the altitude as a line underneath the velocity instead of on the same line.

Also suggest changing `line-height` from 100% to 125% in `.marker p` in main.css to make this two-line layout look better/more spaced out.

```suggestion
                                    '</span>' + 
                                    '<span class="metric">' +
                                        '<br /><span id="altitude-' + shares[user].id + '">0.0</span> ' +
```

Reply:
Very good idea. In fact I tried that first, but the alignment between speed and altitude looked awful. I need to brush up on my CSS skills a bit but I'll come up with something.